app/components/announcements/announcement-card.tsx

```diff
@@ -0,0 +1,49 @@
+import { ViewStyle, Platform, View, Text, TextStyle } from "react-native"
+import TouchableScale from "react-native-touchable-scale"
+import { fontScale, spacing, color } from "../../theme"
+import { openLink } from "../../utils/helpers/open-link"
+
+const ANNOUNCEMENT_CARD: ViewStyle = {
+  maxWidth: "100%",
+  minWidth: "100%",
+  minHeight: 80 * fontScale,
+  backgroundColor: color.inputBackground,
+  marginBottom: spacing[4],
+  paddingTop: spacing[3],
+  paddingBottom: spacing[2],
+  paddingStart: spacing[3],
+  paddingEnd: spacing[4],
+  borderRadius: Platform.select({ ios: 12, android: 8 }),
+  shadowColor: color.palette.black,
+  shadowOffset: { height: 0, width: 0 },
+  shadowOpacity: 0.05,
+  elevation: 1,
+}
+
+const TITLE_STYLE: TextStyle = {
+  fontFamily: "Heebo",
+  color: color.primary,
+  fontSize: fontScale * 16,
```

Comment:
@planecore no need for `* fontScale` since the `Text` component does that automatically when font scaling is on